microprofile/microprofile-rest-client-1.3/src/test/java/org/wildfly/swarm/ts/microprofile/rest/client/v13/MicroprofileRestClient13Test.java

```diff
@@ -0,0 +1,46 @@
+package org.wildfly.swarm.ts.microprofile.rest.client.v13;
+
+import java.io.IOException;
+
+import org.apache.http.client.fluent.Request;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.junit.InSequence;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.wildfly.swarm.arquillian.DefaultDeployment;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(Arquillian.class)
+@DefaultDeployment
+public class MicroprofileRestClient13Test {
+    /**
+     * microprofile-config.properties contains desired key "myClients":
+     * myClients/mp-rest/uri=http://localhost:8080
+     * that is used for @RegisterRestClient(configKey="myClients") ClientAutoCloseable
+     * that is called on url../config-key
+     */
+    @Test
+    @InSequence(1)
```

Comment:
Does the test need to run in a specific sequence? From the casual look, the tests are independent and should not require a specific order.